geoips/plugins/modules/readers/amsr2_netcdf.py

```diff
@@ -324,7 +324,7 @@ def call(
     for fname in fnames:
         # full_xarray = xarray.open_dataset(str(fname))
         full_xarrays = [xarray.open_dataset(str(x)) for x in fnames]
-        full_xarray = xarray.merge(full_xarrays)
+        full_xarray = xarray.concat(full_xarrays, dim="Number_of_Scans")
```

Comment:
I think this is Also a straight up bug fix,  previously did not merge amsr2 swaths correctly?